src/prelude.rs

```diff
@@ -2,7 +2,7 @@
 //!
 //! This includes the system table types, `Status` codes, etc.
 
-pub use crate::{ResultExt, Status};
+pub use crate::{ResultExt, ResultExt2, Status};
```

Comment:
There is one small but useful piece of functionality (treating warnings as errors) that is not supported by the new Results with error payloads. As far as I know, the only way to handle this is to add another extension trait...